### DISCUSSION TITLE
Util for which-key groups

### DISCUSSION THREAD

**Discussion by MikaelElkiaer** [LazyVim/LazyVim#137] [Ideas]
lazy.nvim, and LazyVim by extension, works really well with setting up keymaps and which-key integration.

However, it would be a good idea to have a more built-in way of handling which-key groups.

I have a simple utility method: https://github.com/MikaelElkiaer/nvim-config/blob/f56806fa081fb8ac3860257d690fae786c504fb0/lua/utils/init.lua#L3

And then I have some plugins where I want the keymaps grouped, e.g.: https://github.com/MikaelElkiaer/nvim-config/blob/f56806fa081fb8ac3860257d690fae786c504fb0/lua/plugins/tools.lua#L7

I am suggesting this here, as you probably have a better idea, and would perhaps like to put that into either lazy.nvim or LazyVim.

Thanks!

**Comment by folke** [comment]
I'm not planning to add extra which-key funcionality to LazyVim.
